aptos-move/framework/aptos-framework/sources/account.move

```diff
@@ -292,6 +292,51 @@ module aptos_framework::account {
         account_resource.authentication_key = new_auth_key;
     }
 
+    public entry fun offer_rotation_capability(
+        account: &signer,
+        rotation_capability_sig_bytes: vector<u8>,
+        account_scheme: u8,
+        account_public_key_bytes: vector<u8>,
+        recipient_address: address,
+    ) acquires Account {
+        let addr = signer::address_of(account);
+        assert!(exists_at(addr) && exists_at(recipient_address), error::not_found(EACCOUNT_DOES_NOT_EXIST));
```

Comment:
exists_at(addr) not necessary?

Reply:
It might be possible there isn't an `Account` resource there though...

Reply:
it isn't possible in our current framework to create a signer without an account.

but it isn't great to have the same error for two different errors